test/parallel/test-v8-serdes.js

```diff
@@ -66,6 +66,7 @@ const deserializerTypeError =
 }
 
 {
+  let arrayBufferViewsLength;
```

Comment:
Maybe simplest solution is to add a sentinel:
```js
  let has_writeHostObjectCalled = false;
```
the in `_writeHostObject`
```js
  has_writeHostObjectCalled = true;
```
the first line of `_readHostObject`
```js
  assert.ok(has_writeHostObjectCalled);
```